mlir/include/mlir/Dialect/NVGPU/IR/NVGPU.td

```diff
@@ -728,4 +728,16 @@ def NVGPU_WarpgroupMmaOp : NVGPU_Op<"warpgroup.mma"> {
   let hasVerifier = 1;
 }
 
+def NVGPU_WarpgroupMmaInitAccumulatorOp : NVGPU_Op<"warpgroup.mma.init.accumulator"> {  
+  let summary = "Initializes accumulator matrix for `warppgroup.mma`";
+
+  let description = [{
+    This Op generates and initilizes the accumulator matrix for 
```

Comment:
The typo is still here :).
Did you forget to add this part of the change?

Reply:
Apologize. I am using vscode for github comments. Things are going wrong.